### PR TITLE
Update to read daemon list from package.json

### PR DESCRIPTION
**Pull request by shuse2**
Update to read the package json daemons list to change the execution command.

### CODE REVIEW

File: src/tarballs/bin.ts [hunk 1]
```diff
@@ -35,6 +35,7 @@ if exist "%~dp0..\\bin\\node.exe" (
   }
   const writeUnix = async () => {
     const bin = qq.join([baseWorkspace, 'bin', config.bin])
+    const daemons = config.pjson.lisk?.daemons?.join('|') ?? '';
```
Comment (fchavant) [inline src/tarballs/bin.ts:38]: Do we assume the "daemons" commands contain no `|` or ` `?

File: src/tarballs/bin.ts [hunk 2]
```diff
@@ -83,13 +84,14 @@ else
...
     echoerr ${binPathEnvVar}="\$${binPathEnvVar}" "\$NODE" "\$DIR/run" "\$@"
   fi
-  if [ "$1" = "start" ]; then
-    "$NODE" "$DIR/run" "$@" &
-    cmd_pid=$!
-    wait $cmd_pid
-  else
-    "$NODE" "$DIR/run" "$@"
-  fi
+  case $1 in
```
Comment (fchavant) [inline src/tarballs/bin.ts:87]: We should check if `${daemons} == ''` and if so skip the this block and unconditionnaly run `"$NODE" "$DIR/run" "$@"` instead.

**Review by fchavant — CHANGES_REQUESTED** [review]

File: src/tarballs/bin.ts (hunk 1 above)
Comment (shuse2) [reply src/tarballs/bin.ts:38]: `daemons` is a string array, which should not contain any thing other than command itself

File: src/tarballs/bin.ts (hunk 2 above)
Comment (shuse2) [reply src/tarballs/bin.ts]: Updated to use different logic in case daemons is empty

**Review by fchavant — APPROVED** [review]

**Review by Nazgolze — APPROVED** [review]

**Pull request merged**